src/main/java/org/moss/discord/listeners/AutoModListeners.java

```diff
@@ -0,0 +1,65 @@
+package org.moss.discord.listeners;
+
+import org.javacord.api.DiscordApi;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.MessageAttachment;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+import org.javacord.api.entity.user.User;
+import org.javacord.api.event.message.MessageCreateEvent;
+import org.javacord.api.listener.message.MessageCreateListener;
+import org.moss.discord.Constants;
+
+import java.awt.*;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class AutoModListeners implements MessageCreateListener {
+
+    DiscordApi api;
+    List<String> blacklistedFiles;
+    private Optional<TextChannel> modChannel;
+
+    /*
+    TODO: Add role bypass
```

Comment:
You should probably add a role bypass before merging.

Reply:
TODO: Add role bypass

Reply:
You should probably add a role bypass before merging.